plugin/metrics/disabled/factory.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/spf13/viper"
```

Comment:
`plugin/metrics` is a bad name, we should rename it to `plugin/metricstore`

Reply:
@yurishkuro I can do that in a follow-up PR

Reply:
we can also rename storage/metricsstore to storage/metricstore (single `s`)